From 4cd2da42a7b08a891dcefbe14ea31506aa831328 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Sun, 30 Dec 2018 11:32:51 +0800
Subject: [PATCH] drm: set wait_for_completion_timeout for commit clean up

no need to wait 10000ms for time out. because even the 24fps
panel, the vsync time is 41ms.

Change-Id: I161836eca3f93d954028e06e4175db6c6d7ec734
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/drm_atomic_helper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_atomic_helper.c b/drivers/gpu/drm/drm_atomic_helper.c
index f77d4aa1e58b..a30621abfee1 100644
--- a/drivers/gpu/drm/drm_atomic_helper.c
+++ b/drivers/gpu/drm/drm_atomic_helper.c
@@ -1446,7 +1446,7 @@ void drm_atomic_helper_commit_cleanup_done(struct drm_atomic_state *state)
 		 * before releasing our reference, since the vblank work does
 		 * not hold a reference of its own. */
 		ret = wait_for_completion_timeout(&commit->flip_done,
-						  10*HZ);
+						  msecs_to_jiffies(100));
 		if (ret == 0)
 			DRM_ERROR("[CRTC:%d] flip_done timed out\n",
 				  crtc->base.id);
-- 
2.35.3

